paasta_tools/chronos_serviceinit.py

```diff
@@ -30,23 +30,36 @@
 logging.basicConfig()
 
 
-# Calls the 'manual start' endpoint in Chronos (https://mesos.github.io/chronos/docs/api.html#manually-starting-a-job),
-# running the job now regardless of its 'schedule' and 'disabled' settings. The job's 'schedule' is left unmodified.
-def start_chronos_job(service, instance, job_id, client, cluster, job_config, emergency=False):
+def start_chronos_job(service, instance, job_id, client, cluster, job_config, complete_job_config, emergency=False):
+    """
+    Calls the 'manual start' Chronos endpoint (https://mesos.github.io/chronos/docs/api.html#manually-starting-a-job),
+    running the job now regardless of its 'schedule'. The job's "schedule" is unmodified. If a job is disabled,
+    this function does not do anything.
+    """
     name = PaastaColors.cyan(job_id)
+    # The job should be run immediately as long as the job is not disabled via
+    # the 'disabled' key in soa-configs. Since we clobber the 'disabled' key in
+    # complete_job_config (the dict that gets passed to Chronos), we need to look
+    # at the service's soa-config directly to check if it is disabled.
+    should_run_now = not job_config.get_disabled()
     log_reason = PaastaColors.red("EmergencyStart") if emergency else "Brutal bounce"
```

Comment:
I don't understand why we do this here. 

In fact this all seems really over complicated. Why can't this just be:


```python
if job_config.get_disabled():
    print "You can't emergency start a disabled job. Run `paasta start` first."
else:
    client.run(job_id)
```

All this other stuff is unnecessary (now) imo.